ci-operator/jobs/openshift/release/openshift-release-release-4.3-periodics.yaml

```diff
@@ -4769,6 +4769,88 @@ periodics:
     - name: pull-secret
       secret:
         secretName: ci-pull-credentials
+- agent: kubernetes
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  interval: 8h
+  labels:
+    ci.openshift.io/release-type: informing
+    job-env: openstack
+    job-release: "4.3"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: release-openshift-ocp-installer-e2e-openstack-ppc64le-4.3
+  spec:
+    containers:
+    - args:
+      - --artifact-dir=$(ARTIFACTS)
+      - --give-pr-author-access-to-namespace=true
+      - --input-hash=$(BUILD_ID)
+      - --input-hash=$(JOB_NAME)
+      - --lease-server=http://boskos
```

Comment:
You may want to drop the lease-server (assuming the cloud you're running the tests on has enough quota and capacity) or use a different boskos quota slice for the openstack-ppc64le jobs, otherwise this will interfere with the quota slice for the other openstack jobs:

https://github.com/openshift/release/blob/89bee6a7beb0ef4ea34102329ad671e778f9374d/core-services/prow/02_config/_boskos.yaml#L19-L22